java/dataset/src/test/java/org/apache/arrow/dataset/substrait/TestAceroSubstraitConsumer.java

```diff
@@ -204,4 +205,93 @@ public void testRunBinaryQueryNamedTableNation() throws Exception {
       }
     }
   }
+
```

Comment:
IMO it would be nice to see separate tests for `filter` and `projection` functionality.

Reply:
Added